src/SPIRV/Silk.NET.SPIRVReflect/Enums/AnonymousEnumSpirvReflectL261C1.gen.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+
+using System;
+using Silk.NET.Core.Attributes;
+
+#pragma warning disable 1591
+
+namespace Silk.NET.SPIRVReflect
+{
+    [Flags]
+    [NativeName("Name", "__AnonymousEnum_spirv_reflect_L261_C1")]
+    public enum AnonymousEnumSpirvReflectL261C1 : int
+    {
```

Comment:
Rename anonymous if possible?

Reply:
Yeah the current name is *bad*, but im not sure what would be a better name for this specific enum

Reply:
I wonder if we rename both of these anonymous enums to "Constants" the bakery will kick in?